boltdesignsystem/bolt
# fix: more robust animation resets

fix: more robust animation resets to allow retrigger when no idle or build out is defined

## Asana

https://app.asana.com/0/1126340469288208/1140132371899568/f

## Summary

More robust reset of animations when idle or build out are not defined.

## Details

There was an issue with the radar svg on the first step stage of the demo not retriggering the second time the step became active. We discovered that bolt-animate components that do not have a defined idle or build-out animations were not properly resetting to their initial state. 

## How to test

http://localhost:3000/pattern-lab/patterns/06-experiments-micro-journeys-combos--92-micro-journey-in-themes/06-experiments-micro-journeys-combos--92-micro-journey-in-themes.html

1) Go to the above URL
2) On the first step, the radar svg behind the character on the left should fade in then out
3) Go to another step, then return to the first step. The radar svg should once again fade in then out.
